roles/customize_home/tasks/main.yml

```diff
@@ -3,7 +3,6 @@
   local_action:
     module: stat
     path: "{{ playbook_dir | dirname }}/user_devel_env_files/"
-  become: no
```

Comment:
why change this? https://github.com/theforeman/forklift/commit/308e4b91aa3ef650076db670b2f3582961d91e78

Reply:
it's redundant from a few lines above and ansible was not happy with that.

Reply:
@ekohl Is the issue described and fixed in https://github.com/theforeman/forklift/commit/308e4b91aa3ef650076db670b2f3582961d91e78 no longer a concern?

Reply:
It is, but it wasn't working. 3557d6692858eac4d52273bb8d464880ecc1dc00 added a proper boolean but failed to remove the incorrect one.